documentation/en/default-lotus-config.toml

```diff
@@ -226,14 +226,7 @@
   #
   # type: bool
   # env var: LOTUS_FEVM_ENABLEETHRPC
-  #EnableEthRPC = false
-
-  # EthTxHashMappingLifetimeDays the transaction hash lookup database will delete mappings that have been stored for more than x days
-  # Set to 0 to keep all mappings
-  #
-  # type: int
-  # env var: LOTUS_FEVM_ETHTXHASHMAPPINGLIFETIMEDAYS
-  #EthTxHashMappingLifetimeDays = 0
+  #EnableEthRPC = true
```

Comment:
Do we really want to default to enabling the EthRPC right now?  That seems like an unnecessary change.

Reply:
This is the ongoing debate - @aarshkshah1992 thinks it's best to just turn on indexing by default and let people opt out, because it's got general benefits, but on the other side are the interests of people running trim nodes that just want recent state, don't need eth, and care more about memory footprint and machine resource use than they do about responsiveness. At this stage @aarshkshah1992 is out-voted.

I could imagine a future, where we're indexing even more things to speed stuff up, that it makes sense to always turn it on because it's always a win, but only if we're able to turn certain features off -- events in particular, but also eth address lookups.

Reply:
I agree given the overwhelming support for it and am happy to not change any defaults here. Reverted them all.